README.md

```diff
@@ -1,167 +1,168 @@
 # Awesome Translations [![Awesome](https://awesome.re/badge-flat.svg)](https://awesome.re)
 
-> The transfer of the meaning of a text from one language to another.
+> Restituer le sens d'un texte d'une langue à une autre.
 
-This list contains tools, books, articles, blogs, courses and everything related to translations ([t9n](https://en.wikipedia.org/wiki/Translation) - translation, [l10n](https://en.wikipedia.org/wiki/Language_localisation) - localization, [g11n](https://en.wikipedia.org/wiki/Language_localisation#Globalisation_versus_localisation) - globalization, [m17n](https://www.definify.com/word/multilingualization#:~:text=Noun,into%2C%20or%20for%20multiple%20languages) - multilingualization, [i18n](https://en.wikipedia.org/wiki/Language_localisation#Globalisation_versus_localisation) - internationalization, synonymous with globalization).
+Cette liste contient des outils, ouvrages, articles, blogs, cours et autres ressources en rapport avec la traduction ([t9n](https://fr.wikipedia.org/wiki/Traduction) - traduction, [l10n](https://en.wikipedia.org/wiki/Language_localisation) - localisation, [g11n](https://en.wikipedia.org/wiki/Language_localisation#Globalisation_versus_localisation) - globalisation, [m17n](https://www.definify.com/word/multilingualization#:~:text=Noun,into%2C%20or%20for%20multiple%20languages) - multilinguisme, [i18n](https://en.wikipedia.org/wiki/Language_localisation#Globalisation_versus_localisation) - internationalisation, synonyme de globalisation).
 
-## Contents
+## Contenu
 
-- [Translations](#translations)
-- [Platforms](#platforms)
-- [Apps](#apps)
-- [Tools](#tools)
+- [Traductions](#traductions)
+- [Plateformes](#plateformes)
+- [Applications](#applications)
+- [Outils](#outils)
 - [Articles](#articles)
 - [Blogs](#blogs)
-- [Online Courses](#online-courses)
-- [Books](#books)
-- [Other](#other)
-
-## Translations
+- [Cours en ligne](#cours)
+- [Ouvrages](#ouvrages)
+- [Autres](#autres)
+
+## Traductions
+
+- [Anglais](https://github.com/mbiesiad/awesome-translations)
+- [Polonais](https://github.com/mbiesiad/awesome-translations/tree/pl_PL)
+- [Espagnol](https://web.archive.org/web/20220511155347/https://github.com/JoseDeFreitas/awesome-translations/tree/es_ES)
+- [Italien](https://github.com/niedev/awesome-translations/tree/it_IT)
+- [Français](https://github.com/alexture/awesome-translations/tree/fr_FR)
+
+## Plateformes
 
-- [English](https://github.com/mbiesiad/awesome-translations)
-- [Polish](https://github.com/mbiesiad/awesome-translations/tree/pl_PL)
-- [Spanish](https://web.archive.org/web/20220511155347/https://github.com/JoseDeFreitas/awesome-translations/tree/es_ES)
-- [Italian](https://github.com/niedev/awesome-translations/tree/it_IT)
```

Comment:
One of the solution could be to add as 25 line - [French](https://github.com/alexture/awesome-translations/blob/fr_FR/README.md) and undo other changes in this file (as main file, should be in English).

Reply:
`- [French](https://github.com/alexture/awesome-translations/blob/fr_FR/README.md)`